examples/todo-intermediate/test/acceptance/application.acceptance.ts

```diff
@@ -5,23 +5,31 @@
 
 import {createClientForHandler, expect, supertest} from '@loopback/testlab';
 import {TodoListApplication} from '../../src/application';
-import {Todo} from '../../src/models/';
-import {TodoRepository} from '../../src/repositories/';
-import {givenTodo} from '../helpers';
+import {Todo, TodoList} from '../../src/models/';
+import {TodoRepository, TodoListRepository} from '../../src/repositories/';
+import {givenTodo, givenTodoList} from '../helpers';
 
 describe('Application', () => {
   let app: TodoListApplication;
   let client: supertest.SuperTest<supertest.Test>;
   let todoRepo: TodoRepository;
+  let todoListRepo: TodoListRepository;
 
   before(givenRunningApplicationWithCustomConfiguration);
   after(() => app.stop());
 
   before(givenTodoRepository);
+  before(givenTodoListRepository);
   before(() => {
     client = createClientForHandler(app.requestHandler);
   });
 
+  // TODO: refactor into multiple files
+  beforeEach(async () => {
+    await todoRepo.deleteAll();
```

Comment:
TODO: split this test file into two:
- todo.acceptance.ts
- todo-list.acceptance.ts